### DISCUSSION TITLE
✅ Flip Box Widget

### DISCUSSION THREAD

**Discussion by Andrenalina** [elementor/elementor#21170] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi team!

Flip Box Widget doesn't allow you select the tag you want _(h1, h2, h3, h4, h5, h6, span, div)_ for the title and description fields on both the front and back.

### Describe the solution you'd like

It would be great if in both cases you could select the tag you need (h1, h2, h3, h4, h5, h6, span, div) to define a good hierarchy of headings and benefit SEO. Thanks in advance.

Best regards! :)

**Comment by rami-elementor** [comment]
Fixed in Elementor 3.15.0

Ref: https://developers.elementor.com/elementor-3-15-developers-update/#:~:text=of%20the%20box.-,SEO%20Improvements,-Elementor%20constantly%20strives
